src/test/java/org/jabref/logic/importer/fetcher/IEEETest.java

```diff
@@ -121,9 +121,12 @@ void searchByQueryFindsEntry() throws Exception {
                 .withField(StandardField.PUBLISHER, "IEEE")
                 .withField(StandardField.KEYWORDS, "Portals, Documentation, Computer bugs, Joining processes, Industries, Open source software, Newcomers, Newbies, Novices, Beginners, Open Source Software, Barriers, Obstacles, Onboarding, Joining Process")
                 .withField(StandardField.TITLE, "Overcoming Open Source Project Entry Barriers with a Portal for Newcomers")
-                .withField(StandardField.FILE, ":https\\://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7886910:PDF")
-                .withField(StandardField.ABSTRACT, "Community-based Open Source Software (OSS) projects are usually self-organized and dynamic, receiving contributions from distributed volunteers. Newcomer are important to the survival, long-term success, and continuity of these communities. However, newcomers face many barriers when making their first contribution to an OSS project, leading in many cases to dropouts. Therefore, a major challenge for OSS projects is to provide ways to support newcomers during their first contribution. In this paper, we propose and evaluate FLOSScoach, a portal created to support newcomers to OSS projects. FLOSScoach was designed based on a conceptual model of barriers created in our previous work. To evaluate the portal, we conducted a study with 65 students, relying on qualitative data from diaries, self-efficacy questionnaires, and the Technology Acceptance Model. The results indicate that FLOSScoach played an important role in guiding newcomers and in lowering barriers related to the orientation and contribution process, whereas it was not effective in lowering technical barriers. We also found that FLOSScoach is useful, easy to use, and increased newcomers' confidence to contribute. Our results can help project maintainers on deciding the points that need more attention in order to help OSS project newcomers overcome entry barriers.");
-        List<BibEntry> fetchedEntries = fetcher.performSearch("Overcoming Open Source Project Entry Barriers with a Portal for Newcomers");
+                .withField(StandardField.FILE, ":https\\://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7886910:PDF");
+        // The quotes are important to get the paper found. Without quotes, the fetcher constructs a query making IEEE returning no entry at all.
+        // It seems that IEEE cannot handle more than 6 AND conditions in a query as of 2021-06
+        List<BibEntry> fetchedEntries = fetcher.performSearch("\"Overcoming Open Source Project Entry Barriers with a Portal for Newcomers\"");
```

Comment:
Please add a new test case for the query without quotes.

In the background, without quotes, the query is converted to https://ieeexploreapi.ieee.org/api/v1/search/articles?apikey=...&querytext=Overcoming+AND+Open+AND+Source+AND+Project+AND+Entry+AND+Barriers+AND+with+AND+a+AND+Portal+AND+for+AND+Newcomers&max_records=20&start_record=01

> The reason the query [without quotes] fails is because it contains the stop words `a`, `for` and `with`. The search API has a list of such commonly occurring words that are skipped so that they don't negatively impact relevance ranking - [IEEE doesn't] want documents that have a high frequency of  such words to float to the top of the result set. Such words are only kept when they are within quotes, since in such cases they are important in making up the phrase that the user is searching on.
>
> So - if you search for `and` in Xplore you get 0 results, but `"and"` returns 5m+ results - practically the entire corpus.